apps/ideaspace/src/components/common/BackButton/BackButton.js

```diff
@@ -8,8 +8,10 @@ const BackButton = ({ buttonType, clickHandler, backRoute }) => {
   const goBack = () => {
     if (backRoute) {
       router.push(backRoute);
+    } else if (window.history.length > 1) {
+      window.history.back();
     } else {
-      window.history.back(-1);
+      window.close();
```

Comment:
is closing the current window when users navigate part of the requirements for this change? @Ahsaask @chadcrotchett

Reply:
Requirement: I want to be able to navigate back from the 'terms and conditions' page to the previous page I visited.

We decided that when the user navigates to a new tab for idea-space terms condition, we will close the tab to prompt user to get back to the submission form. Ideally this should not replace any other functionalities. Let me know if there are some concerns.